app/jobs/analytics/identify_user_job.rb

```diff
@@ -0,0 +1,15 @@
+module Analytics
+  class IdentifyUserJob < AnalyticsJob
+
+    def payload(params)
+      {
+        user_id: params['owner']['id'],
+        traits: {
+          email: params['email'] || "",
+          user: params['user'],
+          account: params['account_id'] || ""
```

Comment:
Ok - still working on what these payloads should actually be, waiting on Segment to respond.